pom.xml

```diff
@@ -139,7 +139,6 @@
             <groupId>javax.transaction</groupId>
             <artifactId>jta</artifactId>
             <version>${jta.api.version}</version>
-            <scope>provided</scope>
```

Comment:
Right.  You fixed the underlying issue (it was always adding JTA support).  Now that its only for web modules, bringing in JTA support always doesn't make sense.  Now it should either be provided or compile & optional.    The prior problem was that it was marked as provided, but was required always.